gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/tests/integration/ResourcesTest.kt

```diff
@@ -747,7 +722,8 @@ class ResourcesTest : GradlePluginTestBase() {
         with(
             testProject(
                 "misc/appleResources",
-                defaultTestEnvironment.copy(kotlinVersion = "2.2.0-RC2"))
+                defaultTestEnvironment.copy(kotlinVersion = "2.2.0-RC2")
```

Comment:
Why do we here RC2 instead of release? Why not constant?

Reply:
it is quite old test when this version wasn't released